scripts/fetch_split_data.py

```diff
@@ -0,0 +1,28 @@
+import pandas as pd
+import os
+import click
+from sklearn.model_selection import train_test_split
+from ucimlrepo import fetch_ucirepo
+
+"""
+Usage: python scripts/fetch_split_data.py --output_path="data/processed" --uci_id=109
+"""
+@click.command()
+@click.option('--output_path')
+@click.option('--uci_id', type=int)
```

Comment:
I think it might be better to hardcode this, just because we don't know if it works with other uci datasets (and other datasets may not have a "class" to stratify for the test split)

Reply:
that is very true! will change it!